README.md

```diff
@@ -1,5 +1,93 @@
-# sda
+# Sensitive Data Archive
 
 `SDA` contains all components of [NeIC Sensitive Data Archive](https://neic-sda.readthedocs.io/en/latest/) It can be used as part of a [Federated EGA](https://ega-archive.org/federated) or as a isolated Sensitive Data Archive.
 
 For more information about the different components see the readme files in the respecive folders.
+
+## Developing components of the SDA stack
+
+If you wish to work on the SDA stack itself you'll first need [Go](https://www.golang.org/) installed on your machine.
```

Comment:
Include pointer(s) to where the minimum version to get can be checked? (E.g. Ubuntu 22.04.3 LTS will give you Golang 1.18 if you ask it by `apt install golang` and I think that could cause issues for people).

Reply:
Easy to miss, especially since I don't install from apt.